### PR TITLE
Upgrading to ember-cli@2.16.2

### PR DESCRIPTION
**Pull request by alvincrespo**
## Changes

- Upgrades ember-cli

### CODE REVIEW

**Comment by alvincrespo** [comment]
Please do not merge this in just yet. There seems to be an issue when the app utilizes this update.

**Comment by alvincrespo** [comment]
This should be all set now, tested against master.

File: addon/initializers/custom-warnings.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ export function initialize() {
 
   Ember.computed.mapBy = function(...args) {
     if (parseInt(Ember.VERSION) < 2) {
-      Ember.warn(`"Ember.computed.mapBy" does not always work as expected in this version of ember, write your own computed property instead.\ Property: "${args[0]}`);
+      Ember.warn(`"Ember.computed.mapBy" does not always work as expected in this version of ember, write your own computed property instead. Property: "${args[0]}`);
```
Comment (Nikash) [inline addon/initializers/custom-warnings.js]: Ummm, is there a mismatched quote here?

**Comment by alvincrespo** [comment]
Closing in favor of #16

**Pull request closed without merging**